src/components/rangeslider/attributes.js

```diff
@@ -43,6 +43,7 @@ module.exports = {
     visible: {
         valType: 'boolean',
         dflt: true,
+        role: 'style',
```

Comment:
should be `info`.

By the way, these `role` fields have no effect in plotly.js. They are used in the plotly.py when the [`strip_style`](https://github.com/plotly/plotly.py/blob/2594076e29584ede2d09f2aa40a8a195b3f3fc66/plotly/plotly/plotly.py#L265) option is turned on. They get stripped out in [`compress_attributes.js`](https://github.com/plotly/plotly.js/blob/master/tasks/util/compress_attributes.js).